plugin/cmd/kube-scheduler/app/configurator.go

```diff
@@ -186,3 +191,100 @@ func (sc schedulerConfigurator) Create() (*scheduler.Config, error) {
 
 	return sc.CreateFromConfig(*policy)
 }
+
+func (sc *schedulerConfigurator) SetupPolicyConfigMapEventHandlers(client clientset.Interface, informerFactory informers.SharedInformerFactory) {
+	// selector targets only the scheduler's policy ConfigMap.
+	selector := cache.NewListWatchFromClient(client.CoreV1().RESTClient(), "configmaps", sc.policyConfigMapNamespace, fields.OneTermEqualSelector(api.ObjectNameField, string(sc.policyConfigMap)))
```

Comment:
There is no config file. The new, preferred way of configuring scheduler policies is to use a ConfigMap object. Use of a ConfigMap object let's us use an API call to update scheduler configuration.